projects/plugins/protect/src/js/components/scan-footer/index.jsx

```diff
@@ -125,18 +126,22 @@ const FooterInfo = () => {
 
 const ScanFooter = () => {
 	const { waf } = window.jetpackProtectInitialState || {};
-	return waf.wafSupported ? (
-		<SeventyFiveLayout
-			main={ <ProductPromotion /> }
-			secondary={ <FooterInfo /> }
-			preserveSecondaryOnMobile={ true }
-		/>
-	) : (
-		<Container horizontalSpacing={ 0 } horizontalGap={ 0 } fluid={ false }>
-			<Col>
-				<FooterInfo />
-			</Col>
-		</Container>
+	return (
+		<AdminSectionHero>
```

Comment:
The designs keep the white footer, so I don't believe we need to add the `<AdminSectionHero>` here.

Reply:
The `FirewallFooter` is in a `<AdminSectionHero>` so now that we are matching the structure  on the Scan page (header - grey, body - white, footer - grey), we should maintain styling consistency between the two. I am good either way, but if we opt to have the keep the footer white here as per the designs, I think we should also update the `FirewallFooter`. Thoughts?